src/debug/debug.proto

```diff
@@ -91,6 +92,15 @@ message App {
   repeated Pod pods = 2;
   google.protobuf.Duration timeout = 3;
   Pipeline pipeline = 4;
+  google.protobuf.Any extra_args = 5;
```

Comment:
how about a ProfileArgs field and a LokiArgs field instead of Any?

Reply:
Done.